packages/compass-components/src/components/feedback-popover.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { GuideCue as LGGuideCue } from '@leafygreen-ui/guide-cue';
+
+import { TextArea, css, spacing } from '..';
+
+const guideCueStyles = css({
+  minWidth: spacing[7] * 4,
+});
+
+type LGGuideCueProps = React.ComponentProps<typeof LGGuideCue>;
+
+// Omit the props we are handling.
+export type FeedbackPopoverProps = Omit<
+  LGGuideCueProps,
+  'currentStep' | 'numberOfSteps' | 'children' | 'title'
+> & {
+  onFeedback: (text: string) => void;
```

Comment:
Tiny naming nit: maybe `onFeedbackSubmit` or just `onSubmit`? I think the part about this being feedback can be inferred from the component name